notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>I think you should use subsections to separate the diffente analysis. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Agree. I change the title of this section to clearly explain that I am computing intermediate value that will be used later in the analysis</p>